openwisp_network_topology/tests/test_api.py

```diff
@@ -90,12 +92,12 @@ def test_detail(self):
     def test_list_unpublished(self):
         self._unpublish()
         response = self.client.get(self.list_url)
-        self.assertEqual(len(response.data['collection']), 0)
+        self.assertEqual(len(response.data['collection']), 1)
```

Comment:
we should add a filter that allows this API to work as before, that is, if a topology is not published it shall not be shown,